packages/engine-core/src/binary/Connection.ts

```diff
@@ -54,8 +54,7 @@ export class Connection {
     if (this._pool) return
 
     this._pool = new Pool(url, {
-      connections: 100,
```

Comment:
Before:
* We instantiated up to 100 Clients
* With each socket sending up to 10 requests